specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/Sessions.json

```diff
@@ -0,0 +1,387 @@
+        "operationId": "ContainerAppsSessionPoolSessionsGeneration_ListGenerated",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SessionPoolNameParameter"
+          },
+          {
+            "name": "generateSessionsEnvelope",
+            "in": "body",
+            "description": "Properties used to generate a session",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/GenerateSessionsRequest"
+            }
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SessionCollection"
+            }
+          },
+          "202": {
+            "description": "Generating SessionPool Sessions.",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-examples": {
+          "Generate Sessions": {
+            "$ref": "./examples/Sessions_GenerateSessions.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/sessionPools/{sessionPoolName}/sessions/{name}": {
+      "get": {
+        "tags": [
+          "ContainerAppsSessionPoolSessions"
+        ],
+        "summary": "Get the properties of a Session.",
+        "operationId": "ContainerAppsSessionPoolSessions_Get",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SessionPoolNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the Session.",
+            "required": true,
+            "type": "string",
+            "minLength": 3,
+            "maxLength": 63
+          },
```

Comment:
parameter like SessionPoolNameParameter ?

Reply:
Sure, previously because this is only used here so I didn't parameterize this, but I agree with you that it would be better to do this, thanks.